server/src/triggers/lifecycle.close/post.mjs

```diff
@@ -0,0 +1,25 @@
+/*
+* Copyright 2021 Comcast Cable Communications Management, LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+function post(ctx, params) {
+  ctx.setTimeout(function() {
+    const result = { state: 'inactive', previous: ctx.fireboltCoreSdk.Lifecycle.state() };
```

Comment:
FYI, the Core SDK manages the value of `previous` so you can leave it out if you want.

Reply:
Good call. Have removed 'previous' from both post.mjs files.